src/api/controller/MyController.hpp

```diff
@@ -36,18 +37,18 @@ class MyController : public oatpp::web::server::api::ApiController {
     auto json = oatpp::parser::json::mapping::ObjectMapper::createShared()
                     ->readFromString<oatpp::Object<LoginDto>>(body);
 
+    auto responseDto = LoginResponseDto::createShared();
+
     if (json && json->username && json->password) {
-      if (json->username == "Jan34") {
-        auto responseDto = LoginResponseDto::createShared();
-        responseDto->success = true;
+      bool loginSuccess = getLogin(json->username->c_str(), json->password->c_str());
```

Comment:
Isn't a `json->username` a string? If so we do not need to create another cstring and then create another temporary string in `getLogin`